.devcontainer/README.md

```diff
@@ -0,0 +1,36 @@
+## Requirements
+
+Please ensure that both [Docker](https://docs.docker.com/desktop/) and optionally [Ollama](https://ollama.com/download) are installed and running on the host machine. To configure your IDE, mount your own `.vscode/` or `.cursor/` folders by uncommenting the `mounts` section in `devcontainer.json`.
```

Comment:
"ensure that both X and optionally Y" is a bit contradictory :) I'd rather have hard requirements, and then some section on optional things

Reply:
Good point. Fixed.